views/projects/project_new.ejs

```diff
@@ -20,25 +18,30 @@
               <input type="text" name="project_description" id="project_description" class="form-control" placeholder="Project Description"/>
               <label class="form-label" for="project_description">Project Description</label>
             </div>
-            <div class="card-body form-outline">
-              <input type="text" name="skils" id="skils" class="form-control" placeholder="Skills list"/>
-              <label class="form-label" for="skils">Skills you use for this project</label>
+            <div class="card-body form-outline input-skill-container">
+              <input type="text" name="skils" class="form-control input-skill"/>
             </div>
-            <div class="card-body form-outline">
-              <input type="text" name="skils" id="skils" class="form-control" placeholder="Skills list"/>
-              <label class="form-label" for="skils">Add project Photo</label>
-              <a class="mx-1" href="/project-photos/new">
-                <i class="fa-solid fa-file-circle-plus"></i>
-              </a>
+            
+            <div class="skill-enter">
+
+            </div>
+            <label class="form-label" for="skills">Skills you use for this project</label>
+            <div class="card-body form-outline add-photo-container">
+              
             </div>
+            <a class="add-photo mx-1" href="/project-photos/new">
```

Comment:
Is this path still valid?

Reply:
Oh no, I missed it. Thank you for highlight. WIll update later.